fdbclient/vexillographer/fdb.options

```diff
@@ -192,6 +192,9 @@ description is not currently required but encouraged.
             description="Enable tracing for all transactions. This is the default." />
     <Option name="distributed_transaction_trace_disable" code="601"
             description="Disable tracing for all transactions." />
+    <Option name="transaction_bypass_unreadable" code="700"
+            description="Allows ``get`` operations to read from sections of keyspace that have become unreadable because of versionstamp operations."
```

Comment:
We should document what it actually returns to you when you read these keys.

Reply:
Also, the database defaults usually indicate that this is setting the option on all transactions created by this database; see for example `transaction_size_limit` (though this isn't being followed everywhere). If you did that, then you could potentially add the extended documentation to the transaction option only. I'll leave it up to you if you want to follow that convention or not.